kiwi_post_run

```diff
@@ -91,13 +93,22 @@ echo "name $NAME"
 echo "version $VERSION"
 echo "release $RELEASE"
 echo "source $IMAGE"
+echo "metadata $METADATA"
+
+# Generate metada file for container-feeder
+cat << EOF > $METADATA
+Name: $CONTAINER_RAW
+Tag: $CONTAINER_TAG
+File: $IMAGE
+EOF
```

Comment:
@flavio, @mjura Up to now kiwi supports only a single tag per image. Not sure multi-tagging is possible or feasible somehow using skopeo, I presume not. I'd say we should stay as tight as possible to current skopeo and umoci capabilities in KIWI, to keep the code simple and maintainable.

Also I think the kind of multi-tagging that @flavio suggests could be achieved with the feeder logic or containment logic. E.g. I would assume that the latest tag is applied by the feeder, not kiwi nor containment-rpm-docker logic.